tasks/bake.js

```diff
@@ -483,18 +500,26 @@ module.exports = function( grunt ) {
 				if ( oldValue === undefined ) values[ forEachName ] = oldValue;
 				else delete values[ forEachName ];
 
-				return fragment;
+				content = fragment;
 
 			} else if( !forEachName ) {
 
 				processExtraBake( extraBake, filePath, destFile, values );
 
-				return linebreak + parse( includeContent, includePath, destFile, values );
+				content = linebreak + parse( includeContent, includePath, destFile, values );
 
 			} else {
 
+				content = "";
+			}
+
+			if( assign !== null ) {
+				parentValues[ assign ] = mout.string.ltrim( content );
```

Comment:
We might have to `ltrim` the same amount of white spaces as the first line of the content, right? So that we trim pieces like:
```html
    <div>
        things are here
    </div>
```
We take away 4 spaces on every line and not just the first one. Just a thought. It kind of doesn't matter and kind of does with HTML's silly rules of how white space and line breaks matter.